packages/miniflare/README.md

```diff
@@ -530,11 +530,14 @@ parameter in module format Workers.
 
   - `assetsPath?: string`
 
-  Path to serve Workers assets from. If set, `__STATIC_CONTENT` and
-  `__STATIC_CONTENT_MANIFEST` bindings will be injected into this Worker. In
-  modules mode, `__STATIC_CONTENT_MANIFEST` will also be exposed as a module
-  with a `string`-typed `default` export, containing the JSON-stringified
-  manifest. Note Workers assets files are never cached in Miniflare.
+  Path to serve Workers assets from.
+  Note Workers assets files are never cached in Miniflare.
+
+  - `assetsKVBindingName?: string`
```

Comment:
TODO @CarmenPopoviciu a bit more detail for `assetsKVBindingName` and `assetsManifestKVBindingName`

Reply:
✅